app/controllers/rentals_controller.rb

```diff
@@ -18,6 +18,7 @@ def show
   # GET /rentals/new
   def new
     @rental = Rental.new
+    @users = User.all
```

Comment:
Consideration for refactor: Move this instantiation to a [Filter](http://guides.rubyonrails.org/action_controller_overview.html#filters)